.github/workflows/integration-tests.yaml

```diff
@@ -80,128 +59,122 @@ jobs:
         run: |
           helm uninstall redis -n ${{ env.TYK_OSS_NS }} --timeout ${{ env.TIMEOUT }}
           kubectl delete namespace ${{ env.TYK_OSS_NS }}
+      
+  tyk-stack:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
 
-      # Tyk Stack tests
+      - name: Set up Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.11.2
+
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.4.0
```

Comment:
which version of k8s is used in integration tests? do we know?

Reply:
Good question. I think it uses latest version. Let me confirm.

Reply:
Default version varies based on the action version. I am pinning it to v1.29.
I think it should be enough to test against latest version as we are already smoke testing against different k8s version.

Reply:
👍 i agree. once the new release is cut, we can update from `node_image` field. thanks for confirmation